Sources/Common/Core/Math/index.js

```diff
@@ -1873,6 +1873,43 @@ export function areBoundsInitialized(bounds) {
   return !(bounds[1] - bounds[0] < 0.0);
 }
 
+export function computeBoundsCenter(bounds) {
+  return [
+    (bounds[0] + bounds[1]) / 2,
+    (bounds[2] + bounds[3]) / 2,
+    (bounds[4] + bounds[5]) / 2,
+  ];
+}
+
+export function computeBoundsScale3(bounds) {
```

Comment:
please add comments because it is not clear what this function does from the function name.

Reply:
I find it odd to call it 'scale'

Moreover it seems similar to BoundingBox/index.js `getLengths`. Maybe all those functions could be moved/refactored there.